package.json

```
@@ -2,7 +2,7 @@
 	"name": "folderize",
 	"displayName": "folderize",
 	"description": "Easily convert a Javascript file into a folder with an index file without breaking imports / exports. Supports .js, .jsx, .ts, .tsx.",
-	"version": "0.0.5",
+	"version": "0.0.6",
```

Since you are adding new features, I believe this should be a minor version bump.

```suggestion
	"version": "0.1.0",
```